src/rust/aggregator/src/main.rs

```diff
@@ -195,6 +198,8 @@ async fn main() -> Result<()> {
                     Duration::from_secs(5 * 60),
                 ))))
             }
+            Pipelines::OrderHistory => data.push(Arc::new(Mutex::new(OrderHistory::new(pool.clone())))),
```

Comment:
How about adding in env var logic that handles both `clap` and env var inputs, similar to how `APTOS_NETWORK` is handled above?

Reply:
I did this. You can see above that I slightly reworked how env vars and args work. Both are taken into account, but env vars always takes priority.

Reply:
Per offline discussion, the comma schema is problematic because environment variables in GCP are parsed of form

`APTOS_NETWORK=my_network,DATABASE_URL,PIPELINES=foo,bar,baz`

Hence a different delimiter is necessary

- [ ] Use a different delimiter, `/`, which we know is valid due to its inclusion in paths/urls as env vars
- [ ] Use a special parser/datatype etc. that gets a vector of pipelines from an env var (this applies to `exclude` and `include` options)
- [ ] Include the `no_default` option
- [ ] Make aggregator-specific ones of form `AGGREGATOR_INCLUDE`, `AGGREGATOR_EXCLUDE`, `AGGREGATOR_NO_DEFAULT`